python/lsst/sdm/schemas/dp1.yaml

```diff
@@ -6571,9 +6568,8 @@ tables:
     ivoa:ucd: meta.id;obs.image
     tap:principal: 1
   - name: detector
-    description: Id of the detector where this forced source was measured. Datatype
-      short instead of byte because of DB concerns about unsigned bytes.
-    datatype: long
+    description: Id of the detector where this forced source was measured.
+    datatype: int
```

Comment:
Same comment - should be `short` and not `int`?

Reply:
Same answer, for this table the MySQL type is `INT`.